lib/api/statistics.js

```diff
@@ -0,0 +1,129 @@
+/* global fetch */
+/**
+ * @fileOverview Module for interacting with the statistcs.json endpoint.
+ *
+ */
+
+// Polyfill for fetch that can be removed once browsers adopt.
+require('whatwg-fetch')
+
+const envPrefixMap = {
+  qa: 'qa.api',
+  staging: 'stg.api',
+  production: 'api'
+}
+
+// Default configuration
+const API_VERSION = 5.4
+const MAX_REQUESTED_IDS = 100;
+
+/**
+ * Calls the statistics.json API with provided options. Returns a promise that
+ * is fulfilled with an array of Results from the API or rejected with the error
+ * message.
+ *
+ * Example:
+ *
+ * getStatistics({
+ *  productIds: ['product1', 'product2', 'product3'],
+ *  environment: 'qa',
+ *  key: 'clients_api_key',
+ *  type: 'Reviews',
+ *  filters: {
+ *   ContentLocale: 'en_US'
+ *  }
+ * }).then(results => {
+ *   // Do something with results
+ * }, errorMsg => {
+ *   // Do something with error.
+ * })
+ *
+ * https://developer.bazaarvoice.com/docs/read/conversations/statistics/display/5_4
+ *
+ * @param  {Object} options object that contains options
+ * @param  {Array}  options.productIds - array of product IDs
+ * @param  {Array}  options.environment - the data environment
+ * @param  {Array}  options.key - client's api key
+ * @param  {string}  options.type - Using "Reviews" returns statistics for all
+ *         content, including syndicated content (if enabled on your API key).
+ *         If you only want statistics for reviews you own that were written for
+ *         the products specified, use "NativeReviews" instead.
+ * @param  {Object}  [options.filters] - Filters object. Keyed by filter name.
+ * @return {Promise} a promise that will be resolved with the raw results
+ *                   from the statistics call.
+ */
+function getStatistics (options) {
```

Comment:
Just a thought: since we're using ES6 now, you can use destructuring to make the function signature more explicit. Then we can just let the Javascript engine do this parameter checking for us.
```javascript
function getStatistics ({ productIds, environment, key, type, filters: {} })
```